.circleci/config.yml

```diff
@@ -238,7 +238,7 @@ jobs:
       - run:
           name: test and generate coverage
           command: yarn coverage:lcov
-          no_output_timeout: 15m
```

Comment:
This timed out on me: https://app.circleci.com/pipelines/github/ethereum-optimism/optimism/13108/workflows/164f734f-d1ae-4b40-b473-a624701f9a13/jobs/416089?invite=true#step-107-33